### DISCUSSION TITLE
Are mobile downloads possible?

### DISCUSSION THREAD

**Discussion by Aejor** [itsToggle/plex_debrid#534] [Q&A] [answered]
Sorry if this is a stupid question, but is it possible to download movies and shows to my phone for offline viewing when using Plex Debrid? I am able to stream just fine, but when I try to download something it returns an error message.

**Comment by AndyZaa** [comment]
Great question - I was wondering the same thing as I'm learning my new plex_debrid install.

I downloaded one of my plex_debrid streaming movies. It downloaded just fine and appeared in Downloads. Then I went offline, wifi and cellular, and tested that it played. It did, so the answer is that you can download a stream to play locally offline.

Have you tried multiple files? Perhaps there is something else interfering.

**Reply by Aejor** [reply]
In that case I must have some of my settings messed up because the downloads will either fail instantly citing lack of storage space (when there definitely should be enough) or will be endlessly stuck on the 'Download Initializing' stage. I've played with all the settings in plex_debrid so I suspect the issue lies somewhere in my Plex setup - I'll keep tinkering with it and see what I can find.

**Comment by Aejor** [comment, marked as answer]
I think I've figured it out. My Plex server computer had no storage space left so it wasn't able to transcode the files for my phone
